hivemind/modules/api/models/event.py

```diff
@@ -0,0 +1,56 @@
+"""
+Event Model for API Module
+"""
+from django.core.exceptions import ValidationError
+from django.db import models
+
+
+class Event(models.Model):
+    """
+    This is the Event Model.
+    """
+    name = models.CharField(
+        'Event Name',
+        max_length=255,
+        help_text='This represents the name of the event.'
+    )
+    description = models.TextField(
+        'Event Description',
+        help_text='This represents a (short) description of the event.'
+    )
+
+    start_time = models.DateTimeField(
+        'Start Time',
+        help_text='This is the time when the event starts.'
+    )
+    end_time = models.DateTimeField(
+        'End Time',
+        help_text='This is the time when the event ends.'
+    )
+
```

Comment:
Please add a `created` and `modified` field. The `created` field will store the Data & Time at which the event is created while the `modified` field will store the last time the Event was modified.